Cargo.lock

```diff
@@ -6191,6 +6191,7 @@ dependencies = [
  "tower",
  "tracing",
  "zcash_address",
+ "zcash_primitives 0.13.0",
```

Comment:
```suggestion
 "zcash_primitives 0.14.0",
```

If we use version 0.14.0 this should automatically change at build but if we apply suggestions from github then we should add this one as well.